pkgs/numpy/numpy.yaml

```diff
@@ -30,6 +30,29 @@ build_stages:
       export LAPACK=None
       export BLAS=None
 
+  - when: machine == 'SGIICEX'
+    name: create-site.cfg
+    after: prologue
+    before: install
+    handler: bash
+    bash: |
+      cat > site.cfg << EOF
+      [mkl]
+      library_dirs = /app/intel/compilers/14/mkl/lib/intel64
```

Comment:
Here I'm just following what was  done  for other platforms (linux, cygwin, CrayXE6).  How  do  you suggest that we refactor the platform specific  info?